Week2/homework/4thereadinglist.js

```diff
@@ -0,0 +1,25 @@
+// Keep track of which books you read and which books you want to read!
+//
+// Declare a variable that holds an array of 3 objects, where each object describes a book and has properties for the title (string), author (string), and alreadyRead (boolean indicating if you read it yet).
+const list = [{title: 'Harry', author: 'J.K. Rowling', alreadyRead: false},{title: 'Lord of the Rings', author: 'J.R.R.Tolkien', alreadyRead: true},{title: 'Les Misérables', author: 'Victor Hugo', alreadyRead: false}];
```

Comment:
Indentation here is not the best. It is very hard to read the object. 🙁

Reply:
I though the content was not that important and did not want to turn it into a 15 row text. But I should have done that anyway. 
Sorry for my lazyness  🙁